velox/exec/Unnest.h

```diff
@@ -39,21 +39,41 @@ class Unnest : public Operator {
   bool isFinished() override;
 
  private:
-  // Generate output for 'size' input rows starting from 'start' input row.
-  //
-  // @param start First input row to include in the output.
-  // @param size Number of input rows to include in the output.
+  // Represents the range of rows to process and specify that first and last row
+  // may need to be processed partially.
+  struct RowRange {
+    // First input row to include in the output.
+    vector_size_t start;
+    // Number of input rows to include in the output.
+    vector_size_t size;
+    // First input row start processing from `firstRowStart_`end processing.
```

Comment:
Should the 'firstRowStart_' here be 'start'?
> Processing of the first input row begins at index 'start' and ends at 'firstRowEnd'.

Reply:
No, it is `firstRowStart_`. You can refer to https://github.com/facebookincubator/velox/pull/10711#discussion_r1728698614 to get the detail.